### PR TITLE
Tile Damage Effects

### PR DESCRIPTION
**Pull request by ThatDan123**
image

CL: Readds window damage overlays and knock sound

CL: Adds damage levels for reinforced and normal metal walls

CL: Adds damage levels for normal metal floors

The floor damage stuff needs to be moved to overlays but we need generic sprites that can go over the floors for that.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs [hunk 1]
```diff
@@ -223,10 +223,10 @@ public void Update()
 					return (damage);
 				}
 
-				lock (PresentTiles)
-				{
-					PresentTiles[damageableLayer].TryGetValue(cellPos, out TileLcation);
-				}
+				// lock (PresentTiles)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs:226]: What did this commented code use to do?

Comment (ThatDan123) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs:226]: nothing, so i just commented it out. Probably should just remove it, but i think it was bod that added it so need to ask him what it was going to be used for.

Comment (corp-0) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs:226]: @Bod9001

**Comment by ThatDan123** [comment]
Just found out that multi gas overlays has been broken, wasnt caused by this PR just checked

File: UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs (hunk 1 above)
Comment (Bod9001) [reply UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaTileMap.cs:226]: yeah looks good to get rid of

**Pull request merged**